components/wise/actions/create-quote/create-quote.mjs

```diff
@@ -46,6 +47,14 @@ export default {
       type: "string",
       optional: true,
     },
+    payoutType: {
+      label: "Payout Type",
+      description: "Amount in source currency to be received by the recipient. E.g. `100.00`. Must specify either **Target Amount** or **Source Amount**",
```

Comment:
correct prop description - added info from docs
```suggestion
     description: "Preferred payout method. Default value is `BANK_TRANSFER`. If you are funding the transfer from a Multi Currency Balance, you must set the `payOut` as `BALANCE` in order to get the correct pricing in the quote . By not doing so, it will default to `BANK_TRANSFER` and the fees will be inconsistent between quote and transfer.",
```